src/test/java/com/scottlogic/practice/mealscheduler/ControllerTests/MealControllerTest.java

```diff
@@ -53,12 +49,14 @@ public void GetAllMeals_ShouldReturnListOfMeals() {
     @Test
     public void PostNewMeal_ShouldReturnMealPosted() {
         var meal = testMeals.get(0);
-        when(mealRepo.save(meal)).thenReturn(new Meal(meal));
+        when(mealRepo.save(meal)).thenReturn(Meal.copyOf(meal));
 
         //Act
         var result = mealController.CreateMeal(meal);
 
         //Assert
-        assertEquals(result, meal);
+        assertEquals(result.getName(), meal.getName());
+        assertEquals(result.getTime(), meal.getTime());
+        assertEquals(result.getUser(), meal.getUser());
```

Comment:
I don't mind particularly, is there a reason to test field by field? Does assertEquals not do this check?